scripts/cfos/make_dataset_data_tables.py

```diff
@@ -129,7 +134,10 @@ def load_excel_input(excel, allowed_dataset_cols, allowed_dataset_tables, skipro
     raw_dataset_dict = pd.read_excel(excel, sheet_name=None, skiprows=skiprows, index_col=None, usecols=lambda x: x in allowed_dataset_cols)
     processed_dataset_dict = {k:v for (k,v) in raw_dataset_dict.items() if k in allowed_dataset_tables}
     print("Success: Excel file has been loaded into a dataframe. Note that any columns that are not defined in the schema will be ignored.")
-    print(processed_dataset_dict)
+    for key, value in processed_dataset_dict.items():
+        print("table: " + key)
```

Comment:
do you want to print the table name once for each column? I think you might want to move this line to before this for-loop.

also a good place to use f-strings here (`f"table: {key}"`) and in the next row